.github/workflows/ci-chaincli.yml

```diff
@@ -2,7 +2,13 @@ name: chaincli CI
 
 on:
   push:
```

Comment:
It looks like we don't even run this on push events, should we move the push trigger? In the job below we have this conditional:

```
    if: ${{ github.event_name == 'pull_request' ||  github.event_name == 'schedule' }}
```